package.json

```diff
@@ -14,10 +14,16 @@
     "gtmsportswear",
     "gtm"
   ],
-  "author": "Chris Young",
+  "contributors": [
+    {
+      "name": "Chris Young"
+    }, {
+      "name": "Sam Schneider"
+    }
+  ],
   "files": [
     "src/js-feature-toggle-manager.ts",
-    "src/js-feature-toggle-manager.test.ts",
```

Comment:
Thoughts on removing the `test` file? 

I feel like each package's tests should be passing before a pull request ever gets packaged up. And I don't like cluttering up the consumers, since gtm is already at 208 unit tests.